tests/functional/query-sort-004.phpt

```diff
@@ -26,6 +26,11 @@ var_dump($query);
 
 $cursor = $manager->executeQuery(NS, $query);
 
+/* Numeric keys of stdClass instances cannot be directly accessed, so ensure the
+ * document is decoded as a PHP array.
+ */
+$cursor->setTypeMap(array('document' => 'array'));
+
 foreach ($cursor as $document) {
     echo $document['0'] . "\n";
 }
```

Comment:
Does "document" apply to the root? That looks very wrong.
It didn't before, and I don't think it was part of this ticket either.